src/databricks/labs/ucx/install.py

```diff
@@ -233,12 +234,36 @@ def warehouse_type(_):
             groups_config_args["selected"] = [x.strip() for x in selected_groups.split(",")]
         else:
             groups_config_args["auto"] = True
+
+        instance_profile = None
+        spark_conf_dict = {}
+        if (
+            self._prompts
+            and self._question("Do you need to configure an external Hive Metastore (Glue)", default="no") == "yes"
+        ):
+            logger.info("Setting up an external metastore")
+            instance_profiles = self._instance_profiles()
+            instance_profile = ""
```

Comment:
Is it possible to detect that? Eg seeing the special spark confs in one of the clusters and using it as a template?

Reply:
It's possible. Maybe from the warehouse setting.

Reply:
Let's start with the manual option and then extend to automate it.